src/lib/components/Error/PropValidator.tsx

```diff
@@ -44,5 +56,22 @@ export default function ErrorGenerator(swapWidgetProps: SwapWidgetProps) {
       }
     }
   }, [convenienceFee, convenienceFeeRecipient])
+
+  const { defaultInputAddress, defaultInputAmount, defaultOutputAddress, defaultOutputAmount } = swapWidgetProps
+  useEffect(() => {
+    if (defaultInputAmount && BigNumber.from(defaultInputAmount).lt(0)) {
```

Comment:
What happens if a string is passed instead of a number? Will `BigNumber.from` throw?

Reply:
no, that's supported usage 👌

Reply:
but if `defaultInputAmount` is invalid (eg `abc`) it will throw. I think this is caught by the ErrorBoundary, though.